crypto/asn1/x_int64.c

```diff
@@ -111,12 +117,23 @@ static int uint32_i2c(ASN1_VALUE **pval, unsigned char *cont, int *putype,
         && utmp == 0)
         return -1;
     if ((it->size & INTxx_FLAG_SIGNED) == INTxx_FLAG_SIGNED
-        && (int32_t)utmp < 0)
+        && (int32_t)utmp < 0) {
+        /* i2c_uint64_int() assumes positive values */
+        utmp = 0 - utmp;
         neg = 1;
+    }
 
     return i2c_uint64_int(cont, (uint64_t)utmp, neg);
 }
 
+/*
+ * Absolute value of INT32_MIN: we can't just use -INT32_MIN as it produces
+ * overflow warnings.
+ */
+
+#define ABS_INT32_MIN \
+    ((uint32_t)INT32_MAX + (uint32_t)(-(INT32_MIN + INT32_MAX)))
```

Comment:
> since we no longer do that...

Well, I wouldn't say we actually did it, because it was never executed on non-two's-complement. I mean basically we don't know if we did it. But what we no longer do is *consider* non-two's complement as an option. But as mental exercise on ones'-complement it would have to be INTxx_MAX+0, right?